Controller/RegistrationFOSUser1Controller.php

```diff
@@ -64,13 +65,14 @@ public function registerAction()
                     $this->get('session')->remove('sonata_basket_delivery_redirect');
                     $url = $this->generateUrl($route);
                 } else {
-                    $url = $this->get('session')->get(
-                        'sonata_user_redirect_url',
-                        $this->generateUrl('sonata_user_profile_show')
```

Comment:
That's not right: 
> The default value **if not found**

There is no check, if the value is empty or null. That's why we need this fix.